### ISSUE TITLE
Error while starting errbot for Slack

### ISSUE THREAD

**Issue by coder-kush** [errbotio/errbot#1649]
**Need help with**

- [ ] Running the bot

**Issue description**
So I think I am at the final step and keep getting an error when trying to run my errbot. 

**Environment (please complete the following information):**
- Errbot version: 6.1.9
- OS version: Amazon Linux 2023
- Python version: 3.9.16
- Using a virtual environment: yes
- Using Docker: no

**Additional info**
Error I get is:
```
11:57:05 ERROR    errbot.plugin_manager     Error loading Webserver.
Traceback (most recent call last):
  File "/opt/errbot/virtualenv/lib64/python3.9/site-packages/errbot/plugin_manager.py", line 444, in activate_non_started_plugins
    if not plugin.is_activated:
AttributeError: 'NoneType' object has no attribute 'is_activated'
11:57:05 INFO     errbot.plugin_manager     Activate plugin: Example.
11:57:05 INFO     errbot.core_plugins.wsvie Checking Example for webhooks
11:57:06 WARNING  errbot.core               Some plugins failed to start during bot startup:
Traceback (most recent call last):
  File "/opt/errbot/virtualenv/lib64/python3.9/site-packages/errbot/plugin_manager.py", line 289, in _load_plugins_generic
    plugin_classes = plugin_info.load_plugin_classes(
  File "/opt/errbot/virtualenv/lib64/python3.9/site-packages/errbot/plugin_info.py", line 100, in load_plugin_classes
    spec.loader.exec_module(modu1e)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/errbot/virtualenv/lib64/python3.9/site-packages/errbot/core_plugins/webserver.py", line 9, in <module>
    from OpenSSL import crypto
  File "/opt/errbot/virtualenv/lib64/python3.9/site-packages/OpenSSL/__init__.py", line 8, in <module>
    from OpenSSL import crypto, SSL
  File "/opt/errbot/virtualenv/lib64/python3.9/site-packages/OpenSSL/crypto.py", line 3279, in <module>
    _lib.OpenSSL_add_all_algorithms()
AttributeError: module 'lib' has no attribute 'OpenSSL_add_all_algorithms'
Error: Webserver failed to activate: 'NoneType' object has no attribute 'is_activated'.
```

Any help to make this work will be greatly appreciated. Thank you in advance!

**Comment by kayman-mk** [comment]
`CORE_PLUGINS` set in the config? If so, this is expected and the Bot will work. As far as I remember there is a PR open to fix this.

**Comment by nzlosh** [comment]
You might be encountering this issue.  https://levelup.gitconnected.com/fix-attributeerror-module-lib-has-no-attribute-openssl-521a35d83769?gi=c80288404d83

tldr; update pip inside the virtualenv
`python3 -m pip install --upgrade pip`

**Comment by fernand0** [comment]
I've found the same problem. The solution seems to be to upgrade pyOpenSSL package inside the virtualenv.
    
     ~/.errbot-ve/bin/pip install -U pyOpenSSL

**Comment by sijis** [comment]
Solution noted.
